core/templates/domain/platform_feature/platform-parameter-domain.constants.ts

```diff
@@ -13,24 +13,11 @@
 // limitations under the License.
 
 /**
- * @fileoverview Unit tests for ClientContext.
+ * @fileoverview Constants for platform parameter domain.
  */
 
-import { ClientContext } from
-  'domain/platform_feature/client-context.model';
-
-describe('Client Context Model', () => {
-  it('should create an instance.', () => {
-    const context = ClientContext.create('Web');
-
-    expect(context.platformType).toEqual('Web');
-  });
-
-  it('should convert an instance to a dict.', () => {
-    const context = ClientContext.create('Web');
-
-    expect(context.toBackendDict()).toEqual({
-      platform_type: 'Web',
-    });
-  });
-});
+export const PlatformParameterDomainConstants = {
```

Comment:
Do we really even need this file at all?

Reply:
OK. We should have a tracking issue for that. Can you point to it?

Reply:
Got it. I have created an issue for the same and have assigned myself - #19285.
Thanks